### PR TITLE
feat: Create m4d-system and m4d-blueprints namespaces

### PR DESCRIPTION
**Pull request by tumido**
Resolves: https://github.com/operate-first/support/issues/59

### CODE REVIEW

**Review by HumairAK — APPROVED** [review]

File: cluster-scope/base/namespaces/m4d-blueprints/namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
```
Comment (HumairAK) [inline cluster-scope/base/namespaces/m4d-blueprints/namespace.yaml:5]: Can we get the ticket label like it was done [here](https://github.com/operate-first/apps/blob/master/cluster-scope/base/namespaces/open-aiops/namespace.yaml#L8) Same for the other namespace.

**Review by anishasthana — APPROVED** [review]

File: cluster-scope/base/namespaces/m4d-blueprints/namespace.yaml (hunk 1 above)
Comment (tumido) [reply cluster-scope/base/namespaces/m4d-blueprints/namespace.yaml:5]: Done

**Comment by tumido** [comment]
@HumairAK please review once more :+1:

**Review by HumairAK — APPROVED** [review]

**Pull request merged**